sdk/core/core-http/Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.3 - 2019-12-10
```

Comment:
Is there a plan to release core-http today?
If not, then we should remove the date here